File: app/src/main/java/com/imangazalievm/bubbble/di/userprofile/UserShotsPresenterModule.java
package com.imangazalievm.bubbble.di.userprofile;

import com.imangazalievm.bubbble.di.global.scopes.Presenter;

import dagger.Module;
import dagger.Provides;

@Module
public class UserShotsPresenterModule {

    private final long userId;

    public UserShotsPresenterModule(long userId) {
        this.userId = userId;
    }

    @Provides
    @Presenter
    long provideUserId() {
        return userId;
    }

}

File: app/src/main/java/com/imangazalievm/bubbble/di/userprofile/UserShotsPresenterModule.kt
package com.imangazalievm.bubbble.di.userprofile

import com.imangazalievm.bubbble.di.global.scopes.Presenter
import dagger.Module
import dagger.Provides

@Module
class UserShotsPresenterModule(private val userId: Long) {

    @Provides
    @Presenter
    fun provideUserId(): Long = userId

}